clusters/hive/pools/msp/admins_msp-cluster-pool_rbac.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: msp-cluster-pool
+- apiVersion: user.openshift.io/v1
+  kind: Group
+  metadata:
+    name: msp-pool-admins
+  users:
+  - pmuller
```

Comment:
I don't think I need to be here

Reply:
Hi @petr-muller I have removed you from that file (together with other people suggested by the openshift-ci bot)